src/client/app/frameworks/core.framework/services/localforage.service.ts

```diff
@@ -0,0 +1,12 @@
+import {Injectable} from '@angular/core';
+
+const localForage:LocalForage = require('localforage');
+
+@Injectable()
+export class LocalForageService {
+
+  public instance(): LocalForage {
```

Comment:
Instead of returning a simple instance here though, I would provide an api to work with localForage. Some standard api like `getItem`, `setItem`, etc. This makes providing with other implementations better as well as allows for better unit testing around the implementations, etc.